src/w3c/headers.js

```diff
@@ -588,12 +588,22 @@ export function run(conf) {
 
   hyperHTML.bind(sotd)`${populateSoTD(conf, sotd)}`;
 
-  if (!conf.implementationReportURI && (conf.isCR || conf.isPR || conf.isRec)) {
+  if (!conf.implementationReportURI && conf.isCR) {
     pub(
       "error",
-      "CR, PR, and REC documents need to have an `implementationReportURI` defined."
+      "CR documents must have an [`implementationReportURI`](https://github.com/w3c/respec/wiki/implementationReportURI) " +
+        "that describes [implementation experience](https://www.w3.org/2018/Process-20180201/#implementation-experience)."
     );
   }
+  if (!conf.implementationReportURI && conf.isPR) {
+    pub(
+      "warn",
+      `${conf.specStatus} documents should include an ` +
```

Comment:
We don't need this `conf.specStatus` here I guess.
`conf.isPR` means `conf.specStatus === "PR"`